motif-docs/components/app/app-search.mdx

```diff
@@ -229,7 +229,7 @@ dispatch({ type: 'SET_RESULTS', results: [] })
 break
 case 'Enter': {
 dispatch({ type: 'SET_RESULTS', results: [] })
-window.open(state.results[state.selectedIndex]?.path, '\_self')
+window.open('/docs' + state.results[state.selectedIndex]?.path, '\_self')
```

Comment:
is there a way to do this without hard coding it here? To make it more robust

Reply:
The issue is we're setting the `basePath` to `/docs` in `next.config.js`. 

So if we prefix the return path with `/docs` from the `buildTree` function in `next.config`, then the relative path on the `<a />` element will be `/docs/docs/...`.

I pushed an alternative solution though. Let me know what you think.